e2e/Makefile

```diff
@@ -2,7 +2,8 @@
 
 .PHONY: create-env
 create-env:  ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
-	curl -sfL https://get.k3s.io | sh -s - --docker 
+	curl -sfL https://get.k3s.io | sh -s - --docker
```

Comment:
@pradumnapandit I think that we need to stick to `minikube` since we need to attach disks to a vm in order to run ceph. Please check out this documentation: https://rook.io/docs/rook/v1.11/Contributing/development-environment/?h=minikube#minikube

Reply:
Okay, will try to cover with`minikube`.

Reply:
Added the ENV creation using minikube.